sharness/dependencies/ipfs-update/util/utils.go

```diff
@@ -12,13 +12,14 @@ import (
 	"strings"
 
 	api "github.com/ipfs/go-ipfs-api"
+	config "github.com/ipfs/ipfs-update/config"
```

Comment:
this file should also be moved into a godeps path just like the one above, and then this path will have to be rewritten to the godeps one...